run/helm/snap/templates/config/_task.tpl

```diff
@@ -0,0 +1,19 @@
+---
+max-failures: 10
```

Comment:
Actually, I had a problem making this design decision myself. I understand that usually we use values.yaml file to configure everything that is configurable in the chart. However, in case of Snap, the user needs to configure:
* `snapteld.conf` - that holds configuration of the plugins and snapteld daemon,
* `task.yaml` - that specifies what metrics user wants to collect,
* `values.yaml` - that specifies i.e. what Docker image will be used.

And it seemed to me that letting the user to configure whole raw config files (templates/config/_snapteld.tpl, _task.tpl) ml` would be a better solution than squeezing all of these config files in one `values.yaml` as it would make this file very large and unreadable to the user (mainly because of the presence of sections like `workflow` in `task.yaml` or `control.plugins` in `snapteld.conf`).

Would you prefer me to template `_snapteld.tpl` and `_task.tpl` completely to the `values.yaml`?